sdk/keyvault/keyvault-admin/samples-dev/accessControlHelloWorld.ts

```diff
@@ -19,10 +19,9 @@ import * as dotenv from "dotenv";
 dotenv.config();
 
 export async function main(): Promise<void> {
-  // DefaultAzureCredential expects the following three environment variables:
-  // - AZURE_TENANT_ID: The tenant ID in Azure Active Directory
-  // - AZURE_CLIENT_ID: The application (client) ID registered in the AAD tenant
-  // - AZURE_CLIENT_SECRET: The client secret for the registered application
+  // This sample uses DefaultAzureCredential, which supports a number of authentication mechanisms.
+  // See https://docs.microsoft.com/en-us/javascript/api/overview/azure/identity-readme?view=azure-node-latest for more information
```

Comment:
We should remove language localizations for each of the sample links:
```suggestion
  // See https://docs.microsoft.com/javascript/api/overview/azure/identity-readme?view=azure-node-latest for more information
```

Reply:
Did a bulk find+replace, hopefully I've got them all. Will review and check.